storages/backends/azure_storage.py

```diff
@@ -268,6 +268,15 @@ def get_available_name(self, name, max_length=_AZURE_NAME_MAX_LEN):
         return super().get_available_name(name, max_length)
 
     def exists(self, name):
+        if not name:
+            try:
+                self.service_client.get_container_client(
+                    self.azure_container
+                )
```

Comment:
I believe this can be replaced with `self.client`.

I think it's probably better to just `return True` here `if not name`.

Reply:
You are right, True should be returned if not name.